### ISSUE TITLE
Improve task creation process

### ISSUE THREAD

**Issue by Megha-Dev-19** [NEARBuilders/gateway#472]
While creating the [task](https://www.nearbuilders.org/builddao.near/widget/Index?page=project&id=efiz.near/project/bos-workspace&tab=tasks) I came across few things 

- [ ] In description we should have a markdown editor, to at least support links and user can expand the textarea if needed
- [ ] For task details we should have a multiple textarea which user can expand, may be we can move the checklist to the description part itself like we have in github
- [ ] Support images with description

Thoughts?

**Comment by elliotBraem** [comment]
I agree with all three of these. We can move all of it inside markdown, for sure.

Also -- what do we think about clicking a task showing a quick view sidebar like we do for projects, rather than modal?

<img width="971" alt="Screenshot 2024-07-08 at 6 02 03 PM" src="https://github.com/NEARBuilders/gateway/assets/16282460/8308f90c-121b-4f4c-941d-656930a11be2">

**Comment by Megha-Dev-19** [comment]
> Also -- what do we think about clicking a task showing a quick view sidebar like we do for projects, rather than modal?
@elliotBraem  yeah definitely, it will bring in consistency.
